pmo/order/new/new.js

```diff
@@ -1,3 +1,7 @@
+/**
+ * @module {Module} new <pmo-order-new>
```

Comment:
This comment is for the entire file so that it can tie together the `ViewModel` and the component constructor to this module and correctly nest the documentation.